src/shared-components/forms/index.js

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Typography from '../typography';
+import HelperText, { ErrorItem, HintItem } from './helperText';
+import { FormContainer, InputContainer } from './style';
+import ErrorIcon from '../../svgs/icons/error-icon.svg';
+
+const FormSection = ({
+  children,
+  displayValidation,
+  errorMessage,
+  isFocused,
```

Comment:
i wonder if there's a way to get this value without having the user pass it in from outside of the component